### PR TITLE
Experiment-nightly-2

### PR DESCRIPTION
**Pull request by tzaffi**
Continuing on with the experiment. This time, I'm mixing in more permanent changes of making the calls out to `Makefile` as succint as possible, so that a separate `nightly.yml` 

Also fixing a bug in `nightly.yml` that prevented confirmation that this strategy works.

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -18,8 +18,8 @@ jobs:
         continue-on-error: true
         name: check latest commit is less than a day
         if: ${{ github.event_name == 'schedule' }}
-        run: test -z $(git rev-list  --after="24 hours"  ${{ github.sha }}) && echo "::set-output name=should_run::false"
-
+        run: test -z "$(git rev-list  --after='24 hours'  ${{ github.sha }})" && echo "::set-output name=should_run::false"
+
```
Comment (tzaffi) [inline .github/workflows/nightly.yml:22]:
```suggestion

```

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -18,8 +18,7 @@ jobs:
         continue-on-error: true
         name: check latest commit is less than a day
         if: ${{ github.event_name == 'schedule' }}
-        run: test -z $(git rev-list  --after="24 hours"  ${{ github.sha }}) && echo "::set-output name=should_run::false"
-
+        run: test -z "$(git rev-list  --after='24 hours'  ${{ github.sha }})" && echo "::set-output name=should_run::false"
```
Comment (tzaffi) [inline .github/workflows/nightly.yml:21]:
```suggestion
        run: test -z "$(git rev-list  --after='24 hours'  ${{ github.sha }})" && echo "::set-output name=should_run::false"

```

File: .github/workflows/nightly.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name: "Nightly Build"
 on:
   schedule:
-    - cron: "10 * * * *"
+    - cron: "*/5 * * * *"
```
Comment (tzaffi) [inline .github/workflows/nightly.yml:4]:
```suggestion
    - cron: "10 * * * *"
```

File: .github/workflows/nightly.yml [hunk 4]
```diff
@@ -11,14 +11,18 @@ jobs:
       should_run: ${{ steps.should_run.outputs.should_run }}
     steps:
       - uses: actions/checkout@v2
+
+      - name: current Boston time
```
Comment (bbroder-algo) [inline .github/workflows/nightly.yml:15]: is boston really the city of note
:NYC:

**Review by bbroder-algo — APPROVED** [review]

File: .github/workflows/nightly.yml (hunk 4 above)
Comment (tzaffi) [reply .github/workflows/nightly.yml:15]: Boston is the center of the Algorand universe.

Comment (tzaffi) [inline .github/workflows/nightly.yml:15]:
```suggestion
      - name: Current Boston time
```

**Pull request merged**